yt_dlp/extractor/xfileshare.py

```diff
@@ -68,6 +68,17 @@ class XFileShareIE(InfoExtractor):
     )
 
     _TESTS = [{
+        'url': 'https://uqload.com/nn85ef9zmgzk',
+        'info_dict': {
+            'id': 'nn85ef9zmgzk',
+            'ext': 'mp4',
+            'title': r're:.*',
+            'thumbnail': r're:https://.*\.jpg',
+            'http_headers': {
+                'Referer': 'https://uqload.com/nn85ef9zmgzk'
+            }
+        }
+    }, {
```

Comment:
1. Use non-piracy link
2. Add md5
3. Use actual title
4. `http_headers` is not needed in test